MODULE.bazel

```diff
@@ -80,6 +83,10 @@ maven.install(
       "org.apache.commons:commons-lang3:%s" % commons_lang_version,
       "junit:junit:%s" % junit_version,
       "org.hamcrest:hamcrest-all:%s" % hamcrest_version,
+
+      ## bazel-distribution dependencies
+      "info.picocli:picocli:%s" % picocli_version,
```

Comment:
Why do we need picocli? What does `bazel-distribution dependencies` mean?

Reply:
As the comment says, it is a dependency of `bazel-distribution` rules which are used for generating `pom.xml`.
This doesn't mean that this dependency is added to any of subprojects. But it has to be listed for the rules so that the dependency (and its version) is discoverable.